src/platform_storage_api/admission_controller/api.py

```diff
@@ -0,0 +1,167 @@
+import base64
+import dataclasses
+import json
+import logging
+from enum import Enum
+from typing import Any, Optional
+
+from aiohttp import web
+
+from platform_storage_api.admission_controller.app_keys import VOLUME_RESOLVER_KEY
+from platform_storage_api.admission_controller.volume_resolver import (
+    KubeVolumeResolver,
+    VolumeResolverError,
+)
+from platform_storage_api.config import Config
+
+
+logger = logging.getLogger(__name__)
+
+
+LABEL_APOLO_ORG_NAME = "platform.apolo.us/org"
+LABEL_APOLO_PROJECT_NAME = "platform.apolo.us/project"
+LABEL_APOLO_STORAGE_MOUNT_PATH = "platform.apolo.us/storage.mountPath"
+LABEL_APOLO_STORAGE_HOST_PATH = "platform.apolo.us/storage.hostPath"
```

Comment:
@zubenkoivan two questions here:
- since this can be a JSON, I think we can support multiple mounts?
- WDYT about a name `platform.apolo.us/injectStorage` ?

Reply:
let's use `platform.apolo.us/inject-storage`

Reply:
changed